delphi/GrFN/networks.py

```diff
@@ -378,22 +424,23 @@ def from_python_src(
         json_filename: str,
         stem: str,
         fortran_file: str,
-        save_file: bool = False,
+        mode_mapper_dict: dict,
+        save_file: bool = False
     ):
         """Builds GrFN object from Python source code."""
         asts = [ast.parse(pySrc)]
         pgm_dict = genPGM.create_grfn_dict(
             lambdas_path,
             asts,
             json_filename,
-            {"file_name": f"{stem}.py"}, # HACK
-            fortran_file,
+            mode_mapper_dict,
+            fortran_file
         )
+
+        # {"file_name": f"{stem}.py"}, # HACK
```

Comment:
Is this commented-out line something we can remove as well?

Reply:
Removed.